src/Uno.UI/UI/Xaml/FrameworkElement.Layout.wasm.cs

```diff
@@ -149,13 +151,12 @@ private void InnerArrangeCore(Rect finalRect)
 
 			if (allowClipToSlot && !needsClipToSlot)
 			{
-				if (arrangeSize.Width < _unclippedDesiredSize.Width - SIZE_EPSILON)
+				if (IsLessThanAndNotCloseTo(arrangeSize.Width, _unclippedDesiredSize.Width))
```

Comment:
The `arrangeSize.Width` is not updated here?

Reply:
Nope... there's 2 phases very similar, that's the first phase.